db/migrations/0004_alter_order_user.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.0.2 on 2023-09-04 17:43
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_ticket_movie_session_alter_ticket_order'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='user',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='orders', to=settings.AUTH_USER_MODEL),
+        ),
+    ]

```

Comment:
Include only one migration with your PR, there is no need to have 3 of them here.